vertx-grpc-server/src/main/java/io/vertx/grpc/server/impl/GrpcServerImpl.java

```diff
@@ -59,6 +65,27 @@ public void handle(HttpServerRequest httpRequest) {
   private <Req, Resp> void handle(MethodCallHandler<Req, Resp> method, HttpServerRequest httpRequest, GrpcMethodCall methodCall) {
     GrpcServerRequestImpl<Req, Resp> grpcRequest = new GrpcServerRequestImpl<>(httpRequest, method.messageDecoder, method.messageEncoder, methodCall);
     grpcRequest.init();
+    GrpcAuthenticationHandler authHandler = method.authHandler;
+    if (authHandler != null) {
```

Comment:
here we need to pause the grpc request and resume it after to avoid loosing messages

Reply:
Is this required because in `ìnit` additional handlers are being added?

Reply:
I think you can write a test for this, by sending messages in the client and have an authentication handler that delays the actual auth handler by a delay and check we got all messages on the server